app/src/main/java/org/mozilla/fenix/components/Analytics.kt

```diff
@@ -117,12 +118,15 @@ class Analytics(
                 // from the app unless the user does so from a UI control.
                 // However, the user may have opt-ed out of mako experiments already, so
                 // we should respect that setting here.
-                val enabled = context.settings().isExperimentationEnabled
+                val enabled = context.components.strictMode.resetAfter(StrictMode.allowThreadDiskReads()) {
```

Comment:
I tried reproducing this strict mode violation but can't, even if I execute this at the beginning of `Application.onCreate`. Interesting that the stacktrace we saw doesn't make much sense as we see a call from this lambda directly to `listFiles`, which doesn't exist: https://gist.github.com/travis79/88779741ced2bddf03fe68797c3faa8f

Either way this is OK to land. We're not worried about `context.settings`.